mfr/extensions/tabular/exceptions.py

```diff
@@ -42,3 +47,20 @@ def __init__(self, message, *args, code: int=500, formatting_function: str='', *
         super().__init__(message, *args, code=code, **kwargs)
         self.formatting_function = formatting_function
         self.attr_stack.append([self.__TYPE, {'formatting_function': self.formatting_function}])
+
+
+class FileTooLargeError(TabularRendererError):
```

Comment:
I remember you mentioned in WB but I am not sure about MFR: when an exception is added, should we add it to somewhere in tests to make sure  thatserialization works?

Reply:
Not needed for MFR.  That test is to make sure the error survives the pickling/unpickling done by WB's celery worker.